ui/js/models/perf/alerts.js

```diff
@@ -116,7 +116,7 @@ treeherder.factory('PhAlerts', [
             ).reverse();
 
             const formatAlert = function (alert, alertList) {
-                return _.padStart(alert.amount_pct.toFixed(0), 3) + '%  ' +
+                return ((alert.amount_pct.toFixed(0)).padStart(3)) + '%  ' +
                 _.padEnd(alert.title, _.max(alertList, function (alert) { return alert.title.length; }).title.length + 5) +
```

Comment:
padEnd is registered in Bug no: 1476654 since every lodash function is filed as the unique bug in Bugzilla. 
For more information, take a look at this: https://bugzilla.mozilla.org/show_bug.cgi?id=1466494 

Moreover, for each bug, a unique PR is made. Refer Ed Morley's comment at https://github.com/mozilla/treeherder/pull/3799 

Regarding extra parentheses, it's always safe to have those extra parentheses when we gonna produce the final return value which comprises of many things that have to be concatenated.

Reply:
The comment in the other PR is for a different case - two unrelated lodash methods being adjusted, in two separate files. In this case, since both `padStart()` and `padEnd()` are so similar in functionality (and thus no extra effort to review/check implementation details), and on the same line, it would probably be best to combine them.

Reply:
I think @edmorley was saying that you shouldn't have one PR that covers multiple bugs. It is however ok to have one bug which covers multiple lodash functions. :) Just trying to save a little extra work for both you and whoever reviews this stuff.

Yes, the parentheses are harmless and "safe", but also unnecessary. It is ok to leave them out.

Reply:
Sure. Then, I will commit for padEnd() conversion to es6, when changes in lodash function `_.max()` has been committed by Eliott Vincent.

Reply:
If you want me take up `_.max()`, I will push the cumulative changes along with that soon. Please do let me know.

Reply:
@Soundharyaam `_.max()` has been taken up in #3858

Reply:
@SirR4T No problem. Carry on.

Reply:
@Soundharyaam @wlach : _.padEnd has been fixed in #3857 , and merged. This PR will need to be rebased...